packages/paste-core/components/callout/src/Callout.tsx

```diff
@@ -115,6 +146,14 @@ export const Callout = React.forwardRef<HTMLDivElement, CalloutProps>(
         <Box display="flex" flexDirection="column" rowGap="space50" flex="1">
```

Comment:
The variant icons should stack about the Callout text.

Reply:
Oh man, no idea how I missed that. Sorry!! addressed in latest commits